src/riscv-integration.adoc

```diff
@@ -973,11 +994,31 @@ is the cause of the fault. The possible CHERI types and causes are encoded as
 shown in xref:mtval-cheri-type[xrefstyle=short] and
 xref:mtval-cheri-causes[xrefstyle=short] respectively.
 
+[#stid,reftext="stid"]
+==== Supervisor Thread Identifier (stid)
+
+The <<stid>> register is an SXLEN-bit read-write register. It is used to
+identify the current thread. The reset value of this register is
```

Comment:
```suggestion
identify the current user-mode thread. The reset value of this register is
```
Since the value is shared we should be explicit that it's for U mode. If S-mode wants to reuse the same ID that's fine too, but maybe it should be something like `sutid` to indicate that it's the writable version of `utid`?